### PR TITLE
removed mineral water brands

### PR DESCRIPTION
**Pull request by stephanegigandet**
CIQUAL has categories for individual brands (e.g. Perrier), they don't really make sense as categories for us, and their Agribalyse impact is the same anyway as they were not evaluated individually.

### CODE REVIEW

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -6799,777 +6799,6 @@ nl:Natuurlijke mineraalwaters, Natuurlijk mineraalwater
...
-nl:Met koolzuur versterkte mineraalwaters, Met koolzuur versterkt mineraalwater
-
-<en:Carbonated natural mineral waters
-en:Mildly carbonated natural mineral water
-de:natürliches Mineralwasser mit wenig Kohlensäure versetzt, natürliche Mineralwässer mit wenig Kohlensäure versetzt
-fi:kevyesti hiilihapotettu luonnollinen kivennäisvesi
-fr:Eau minérale naturelle légèrement gazéifiée
-it:Acqua minerale naturale mediamente carbonata
-nl:Met weinig koolzuur versterkte mineraalwaters, Met weinig koolzuur versterkt mineraalwater
-
```
Comment (svensven) [inline taxonomies/categories.txt:7572]: are the last 2 deletions intentional? they're not brands.

Comment (stephanegigandet) [reply taxonomies/categories.txt:7572]: good  catch, I'll put them back

**Comment by teolemon** [comment]
I kind of told @TaciteOFF and @moon-rabbitOFF that we'd think about a wide-ranging solution, but it's a good start 👍

**Review by teolemon — APPROVED** [review]

**Comment by aleene** [comment]
You might need to add the brand minerals as synonyms, or you will lose the ingredients recognition.

**Comment by stephanegigandet** [comment]
> You might need to add the brand minerals as synonyms, or you will lose the ingredients recognition.

Hi @aleene Sorry I don't understand, can you elaborate?

**Pull request merged**